data/human/human missions.txt

```diff
@@ -7455,3 +7455,573 @@ mission "Adelita 5"
+				accept
+	on accept
+		payment -1000000
+	on complete
+		conversation
+			`You land on <destination> and follow Howard's instructions to the biometric scan center. You hand in the first set of paperwork you filled out, and after scanning your fingerprint, retinas, and getting a blood sample, Howard comes over and says, "Excellent, only one last set of paperwork left to sign." He places one more set of documents on the table and taps the stack of papers.`
+			`	"New regulations dictate that we inform you that the proceeds of this bond may be used in high risk activities such as strip mining, clearing endangered habitats, or doing business on worlds with alleged human rights violations. All boilerplate language, of course, but you have to disclose every possibility these days." He shrugs.`
+			`	He then taps the documents the documents again and says, "This is your final decision and set of paperwork. I would also like to disclose that your initial deposit covering the full one million credits is non-refundable. So, as a last formality, would you like to invest in the full investment suite?"`
+			choice
+				`	"Of course."`
+					goto high
+				`	"I'm backing out. I can't ethically invest in this."`
+			`	Howard nods. "I get it, ethics are important and I respect your decision. Unfortunately, the initial deposit is non-refundable, nothing I can do there. Best of luck finding another investment that suits you."`
+				decline
+			label high
+			action
+				"salary: Syndicate Business" = 500
+			`	"Excellent. As a bond owner of Megaparsec Incorporated, you will now receive a daily dividend. I have some work to attend to with our major shareholders, but it was a pleasure doing business with you, Captain."`
+
+# Part 2
+# Best case: 10m, 2700 per day best case out of the negative after almost 10.1 years
+# Worst case: 11m, 1500 per day worst case out of the negative after 20 years
+# Super Worst case: 11m, 1500 per day worst case out of the negative after 38 years
+# Driven by if you were mean to Howard, which of 3 investment options picked, and if you pay off debt ASAP
+
+mission "Syndicate Business 3"
+	name "Visit Howard on <planet>"
+	description "Howard, from Megaparsec Inc., wants to tell you about an investment opportunity."
+	source
+		government "Syndicate"
+		attributes "core"
+		not planet "Hephaestus"
+	destination "Hephaestus"
+	to offer
+		has "Syndicate Business 2: done"
+		random < 40
+		credits > 40000000
+	on offer
+		conversation
+			`After an uneventful trip to the spaceport you return to the <ship> and see a message from Howard, the Megaparsec salesman who sold you a one million credit bond. The message says:`
+			`	"<first> <last>! My bank contact tells me you've been doing quite well for yourself, and I've come across an even bigger investment opportunity than last time! Meet me at my new office on <planet> (note the address, I've moved up since my promotion). Looking forward to chat!"`
```

Comment:
> Looking forward to chat

is technically incorrect. Maybe you could do this instead?
```suggestion
			`	"<first> <last>! My bank contact tells me you've been doing quite well for yourself, and I've come across an even bigger investment opportunity than last time! Meet me at my new office on <planet> (note the address, I've moved up since my promotion). Looking forward to chatting with you!"`
```